docs/src/symbolic_units.md

```diff
@@ -31,3 +31,9 @@ To convert a quantity in regular base SI units to corresponding symbolic units,
 ```@docs
 uconvert
 ```
+
+In order to obtain the numerical value of the quantity in its SI base units, use `ustripexpand`.
+
+```@docs
+ustripexpand
+```
```

Comment:
I think it would be better to just add this immediately under `uexpand`. No need for the extra sentence